### ISSUE TITLE
IE11 not supported in 1.3

### ISSUE THREAD

**Issue by bryanhickerson** [ember-infinity/ember-infinity#347]
I recently upgraded to 1.1 from 1.0.2 and found out that my app broke in IE11. It looks like it's due to the usage of https://github.com/snewcomer/intersection-observer-admin, which I see explicitly says it does not support IE. Is it expected that ember-infinity no longer supports IE11? It seems worth calling this out in the documentation if that's the case. It's also unfortunate, because that means I'll be stuck on 1.0.2.

**Comment by snewcomer** [comment]
@bryanhickerson What was the error/stack trace you saw in IE11?  

The upgrade to `1.1` was actually removing a long deprecated route mixin [here](https://github.com/ember-infinity/ember-infinity/pull/301/files).  We use a `Service Component` approach [now](https://github.com/ember-infinity/ember-infinity#service-component-approach).  

Did you perhaps install `1.1.1`?  If so, that is when I introduced a static administrator for IntersectionObserver.  However, it is not until 1.3 that I introduce the library you pointed out above. 
 Moreover, if IO is not found, it falls back to [rAF](https://github.com/snewcomer/raf-pool), which is supported.  A few projects I am on use `ember-in-viewport`, which when tested on IE11, work fine.

Could you try a few things if you don't mind?  Could you try v`1.3`?  It is possible that I released a version to fix some bug in IE11 since `1.1`.  If not, could you post the error you see and perhaps I could trace it down?  

Thanks for submitting this issue!

**Comment by bryanhickerson** [comment]
It looks like intersection-observer-admin is coming ember-in-viewport.
```
$ yarn add ember-infinity@1.1.3
yarn add v1.10.1
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning "ember-cli-stylelint > broccoli-stylelint > stylelint > postcss-html@0.23.7" has incorrect peer dependency "postcss-syntax@^0.10.0".
warning "ember-cli-stylelint > broccoli-stylelint > stylelint > postcss-markdown@0.23.7" has incorrect peer dependency "postcss-syntax@^0.10.0".
[5/5] 📃  Building fresh packages...

success Saved lockfile.
success Saved 4 new dependencies.
info Direct dependencies
└─ ember-infinity@1.1.3
info All dependencies
├─ ember-in-viewport@3.2.0
├─ ember-infinity@1.1.3
├─ intersection-observer-admin@0.0.2
└─ raf-pool@0.0.1
```

I'll dig up the stack trace I was seeing. I was actually having a different issue with 1.2.6, where the loading indicator would always be visible when the infinity model was only a single page. I'll try out 1.3 and see if I still see either issue.

**Comment by snewcomer** [comment]
@bryanhickerson That was probably my liberal use of "master" as a result of testing out the perf improvements from ember-in-viewport via the IO administrator in this library 😞.  I'm sorry about that.

https://github.com/ember-infinity/ember-infinity/blob/73b398443de531caa9203b04a4ca11aee27cd623/package.json#L30

I'd be interested to hear if `1.3` or `1.2.7` is not working on IE11.  There may be some fixes I can try out if so.

**Comment by bryanhickerson** [comment]
I tried 1.3 and I didn't have the issue I mentioned with the loading indicator showing for one page of results. However, I still have the IE11 issue. Here is a the stack trace:
```
"SyntaxError: Syntax error
   at ./node_modules/intersection-observer-admin/dist/intersection-observer-admin.es5.js (https://04aa06b6.ngrok.io/frontend/assets/vendor-6f8667166ac4903eb5f5145a6cb542c3.js:233415:7)
   at __webpack_require__ (https://04aa06b6.ngrok.io/frontend/assets/vendor-6f8667166ac4903eb5f5145a6cb542c3.js:233644:4)
   at Anonymous function (eval code:21:55)
   at Module.prototype.exports (https://04aa06b6.ngrok.io/frontend/assets/vendor-6f8667166ac4903eb5f5145a6cb542c3.js:111:5)
   at Module.prototype._reify (https://04aa06b6.ngrok.io/frontend/assets/vendor-6f8667166ac4903eb5f5145a6cb542c3.js:148:7)
   at Module.prototype.reify (https://04aa06b6.ngrok.io/frontend/assets/vendor-6f8667166ac4903eb5f5145a6cb542c3.js:135:7)
   at Module.prototype.exports (https://04aa06b6.ngrok.io/frontend/assets/vendor-6f8667166ac4903eb5f5145a6cb542c3.js:109:5)
   at Module.prototype._reify (https://04aa06b6.ngrok.io/frontend/assets/vendor-6f8667166ac4903eb5f5145a6cb542c3.js:148:7)
   at Module.prototype.reify (ht"
```

**Comment by snewcomer** [comment]
@bryanhickerson Hm.  This is a really good find.  I'm guessing the `module` entry is being overridden by the `main` entry.  I think I can just specify a `main` entry.  One sec.

**Comment by snewcomer** [comment]
@bryanhickerson Would you mind overriding your `ember-in-viewport` dep with master of that addon in your project?  If IE11 if fixed, I'll publish a patch release!

**Comment by bryanhickerson** [comment]
I'm still hitting the same error in IE11 with the new ember-in-viewport:

yarn.lock:
```
ember-in-viewport@~3.2.0:
  version "3.2.1"
  resolved "https://registry.yarnpkg.com/ember-in-viewport/-/ember-in-viewport-3.2.1.tgz#df17dc6b0a3de726280d8dd53e512bb883542938"
  integrity sha512-JTA4bzOYLxX6/ZckKPgvt5i6Wa3kqRR/Tt3k4UiBykDw536Z8BcPrqtIKTwUQQ6wDM2e2Qifpm8MpktelJftVg==
  dependencies:
    ember-auto-import "^1.2.15"
    ember-cli-babel "^6.16.0"
    intersection-observer-admin "0.0.5"
    raf-pool "0.0.4"

ember-infinity@^1.3.0:
  version "1.3.0"
  resolved "https://registry.yarnpkg.com/ember-infinity/-/ember-infinity-1.3.0.tgz#a57424af2ee00df34c9a419373ad105c351bbc12"
  integrity sha512-TU5Z5OQ9rsf53mnSMeTQ/Dv/Nm573kzjzlHczGRRvQp7cy0BUr7+b5GHsEOqa4HIVTPPnVaQf4cc9/qWzOR/eg==
  dependencies:
    ember-cli-babel "~7.1.2"
    ember-in-viewport "~3.2.0"
```

**Comment by snewcomer** [comment]
@bryanhickerson Thx for testing.  I'll have to get a windows env up and running on Monday.

I'm guessing the only visible error that changed for you was was the extension `umd.js`, not the `es5.js`?  That isn't an option anymore.

```
at ./node_modules/intersection-observer-admin/dist/intersection-observer-admin.umd.js
```

The main change I made in a IO pkg was [here](https://github.com/snewcomer/intersection-observer-admin/blob/master/tsconfig.json#L5) to compile to ES5 format instead of ES2015.  This is the [dist](https://gist.github.com/snewcomer/c026f325cb4812a8e1daf622f611a305) and am not able to find where this is failing in IE11.  Do you have any thoughts here?

**Comment by snewcomer** [comment]
@bryanhickerson Does your `config/targets` include 'ie 11'?  I just tested out the prod dist for ember-in-viewport in ie11 and it seems to work with the latest master (including the intersection-observer-admin).

**Comment by bryanhickerson** [comment]
This is my targets:
```
module.exports = {
  browsers: [
    '> 1%',
    'last 2 Chrome versions',
    'last 2 Firefox versions',
    'last 2 Safari versions',
    'IE >= 11'
  ]
};
```

**Comment by snewcomer** [comment]
Woops.  Published `1.3.1` although this issue might be still pertinent.

**Comment by bryanhickerson** [comment]
@snewcomer I just tried the new release and the issue is fixed! I'm seeing unexpected loading indicators again, but I can file a different ticket for that.
